configs/compliance.json

```diff
@@ -1,8 +1,8 @@
 {
   "roles": [
     {
-      "name": "Compliance Access",
-      "description": "An all access role which grants read and write permissions.",
+      "name": "Compliance administrator",
+      "description": "Perform any available operation against any Compliance resource.",
       "system": true,
       "platform_default": true,
       "version": 5,
```

Comment:
The version number should be updated in order to update the seeded roles

Reply:
Doh! Thanks, @astrozzc! 😄 We should create a pre-commit hook to automatically bump any of the config files that have changed. 😉